CRM/Contribute/Form/UpdateBilling.php

```diff
@@ -198,32 +197,33 @@ public function postProcess() {
       list($key) = explode('-', $key);
       $processorParams[$key] = $val;
     }
-    $processorParams['state_province'] = CRM_Core_PseudoConstant::stateProvince($params["billing_state_province_id-{$this->_bltID}"], FALSE);
-    $processorParams['country'] = CRM_Core_PseudoConstant::country($params["billing_country_id-{$this->_bltID}"], FALSE);
+    $processorParams['billingStateProvince'] = $processorParams['state_province'] = CRM_Core_PseudoConstant::stateProvince($params["billing_state_province_id-{$this->_bltID}"], FALSE);
```

Comment:
See https://github.com/civicrm/civicrm-core/pull/21583. billingStateProvince got missed originally from propertyBag but should be there.